app/src/main/java/net/christianbeier/droidvnc_ng/Defaults.kt

```diff
@@ -40,6 +40,10 @@ class Defaults {
         private const val PREFS_KEY_DEFAULTS_ACCESS_KEY = "defaults_access_key"
     }
 
+    @EncodeDefault
+    var listenInterface = "0.0.0.0"
```

Comment:
Comment whether this be an IP address or an interface name.

Reply:
Since we're aiming for interface names, the idea is that no interface should be named "0.0.0.0", basically, this should represent a string that simply states "All interfaces" (I know, quite confusing).
Do you have a better idea for this? I don't like it too much myself, except that... I don't have a better one.